go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.14.0
 	github.com/jfrog/terraform-provider-shared v1.8.0
 	golang.org/x/exp v0.0.0-20220613132600-b0d781184e0d
-	golang.org/x/text v0.3.7
+	golang.org/x/text v0.3.8
```

Comment:
We are upgrading to latest version in #105 here: https://github.com/jfrog/terraform-provider-xray/pull/105/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R14